db/setup/schema.sql

```diff
@@ -1,24 +1,24 @@
 CREATE TABLE
-  IF NOT EXISTS LocalFile(
-    "id" UUID DEFAULT gen_random_uuid() PRIMARY KEY,
+  IF NOT EXISTS LocalFile (
+    "id" UUID DEFAULT gen_random_uuid () PRIMARY KEY,
     "filename" VARCHAR(100) NOT NULL UNIQUE,
     "path" VARCHAR(255) NOT NULL UNIQUE,
     "mimetype" VARCHAR(255) NOT NULL,
     "size" BIGINT NOT NULL,
-    "createdAt" TIMESTAMPTZ NOT NULL DEFAULT NOW()
+    "createdAt" TIMESTAMPTZ NOT NULL DEFAULT NOW ()
   );
 
 CREATE TABLE
-  IF NOT EXISTS Users(
-    "id" UUID DEFAULT gen_random_uuid() PRIMARY KEY,
+  IF NOT EXISTS Users (
+    "id" UUID DEFAULT gen_random_uuid () PRIMARY KEY,
     "username" VARCHAR(20) NOT NULL UNIQUE,
     "email" VARCHAR(50) NOT NULL,
     "fullName" VARCHAR(255) NOT NULL,
     "password" TEXT,
-    "avatarId" UUID REFERENCES LocalFile(id) UNIQUE,
+    "avatarId" UUID UNIQUE REFERENCES LocalFile (id) ON DELETE SET NULL,
```

Comment:
we plan to make this mandatory (not null) on https://github.com/42AGV/ft_transcendence/issues/252#event-7648782981. Because of having add default avatars (and also removing the fallback url we have on webapp) all users should now always have an avatar

Reply:
Good. Will merge when it's on master. Also, this is actually taken from the block MR. Not particularly attached to these changes.